<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeAtlas</string>
  </property>
  <widget class="CodeAtlas::CodeView" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowVariables"/>
   <addaction name="actionShowFunctions"/>
   <addaction name="actionShowNames"/>
   <addaction name="actionShowWordCloud"/>
  </widget>
  <action name="actionShowVariables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showVariables</string>
   </property>
  </action>
  <action name="actionShowFunctions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showFunctions</string>
   </property>
  </action>
  <action name="actionShowNames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showNames</string>
   </property>
  </action>
  <action name="actionShowWordCloud">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showWordCloud</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeAtlas::CodeView</class>
   <extends>QWidget</extends>
   <header location="global">codeview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
